Errors while downloading mcumgr

Hello everyone.
I'm trying to get mcumgr using a go command but I get such errors :+1: 

```sh
$ go get github.com/apache/mynewt-mcumgr-cli/mcumgr
# mynewt.apache.org/newtmgr/newtmgr/cli
go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:102:22: undefined: ishell.Context
go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:144:16: undefined: ishell.Context
go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:168:21: undefined: ishell.Context
go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:191:23: undefined: ishell.Context
go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:255:22: undefined: ishell.Context
go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:267:16: undefined: ishell.Context
go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:321:17: undefined: ishell.Context
go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:375:19: undefined: ishell.Context
go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:417:24: undefined: ishell.Context
go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:444:7: shell.AddCmd undefined (type *ishell.Shell has no field or method AddCmd)
go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:444:7: too many errors
```

I've never had those errors before. Any suggestion where should I find a bug?

```
lawrence@Fred:~$ mkdir foo
lawrence@Fred:~$ cd foo
lawrence@Fred:~/foo$ export GO111MODULE=on
lawrence@Fred:~/foo$ go get github.com/apache/mynewt-mcumgr-cli/mcumgr
# mynewt.apache.org/newtmgr/newtmgr/cli
../.go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:134:18: undefined: ishell.Context
../.go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:164:16: undefined: ishell.Context
../.go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:193:21: undefined: ishell.Context
../.go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:249:23: undefined: ishell.Context
../.go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:304:22: undefined: ishell.Context
../.go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:326:16: undefined: ishell.Context
../.go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:356:17: undefined: ishell.Context
../.go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:386:19: undefined: ishell.Context
../.go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:409:24: undefined: ishell.Context
../.go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:439:7: shell.AddCmd undefined (type *ishell.Shell has no field or method AddCmd)
../.go/src/mynewt.apache.org/newtmgr/newtmgr/cli/interactive.go:439:7: too many errors
lawrence@Fred:~/foo$ go version
go version go1.10.4 linux/amd64
```

--- a/go.mod
+++ b/go.mod
@@ -4,5 +4,5 @@ go 1.12
 
 require (
 	mynewt.apache.org/newt v0.0.0-20190723172213-9f7829a76fef
-	mynewt.apache.org/newtmgr v0.0.0-20190801165443-4ba7e39aca9f
+	mynewt.apache.org/newtmgr v0.0.0-20190808223606-13b828d6f85f
 )

